spec/services/work_packages/set_attributes_service_spec.rb

```diff
@@ -145,6 +145,68 @@
     it_behaves_like 'service call'
   end
 
+  context 'for done_ratio' do
```

Comment:
I find it strange to have the implementation and the spec in 2 different commits. Didn't you write them at the same time?

Reply:
I don't recall why. I probably didn't finish late one afternoon, committed, then continued with missing specs the next day. I'll squash them